Dockerfile

```
@@ -1,11 +1,15 @@
-FROM node:6-alpine
+FROM node:8-alpine
 
 RUN apk update; apk add ncurses make bash
```

```suggestion
RUN apk update; apk add --no-cache --virtual ncurses make bash .gyp python g++
```
Is there anything that prevents us from doing this? (obviously, you will also have to remove the followin line)
In general, it is better to squash docker commands that belong to the same logical task, as this creates less layers.

---

You are right, it is now as you suggest it. I have also done the same change at the frontend now.